### ISSUE TITLE
Change colors for a11y

### ISSUE THREAD

**Issue by zepumph** [phetsims/friction#41]
From accessibility meeting today, many think that if we change the colors, it will help to bring out the pieces of the sim we really want to. In general we care about the atoms in the magnified view, and the focus highlights of the sim. @amanda-phet volunteered to help out with the color redesign.

**Comment by amanda-phet** [comment]
In particular, the yellow was so bright that the bright pink didn't stand out against it, so we've made it a soft blue. The green is the same.

The two shades of blue in this mockup are:
#7de2f9 for the book
#00ffff for the atoms 

<img width="573" alt="screen shot 2017-09-19 at 1 07 15 pm" src="https://user-images.githubusercontent.com/7852916/30610580-de328e12-9d3b-11e7-9d12-1433feee3b75.png">

**Comment by amanda-phet** [comment]
I should also add that we changed the outline from red to black, and I changed the stroke around the books to be gray. It might help to see how it looks in action, but hopefully this way the black dashed lines won't conflict with the pages and outlines in the books.

**Comment by zepumph** [comment]
@amanda-phet here is a first pass. Let me know what you think! I looked over the list from the meeting and I think I covered all the color stuff. Some of the focus highlight is still wonky. But there are other issues for that.

**Comment by amanda-phet** [comment]
I just looked on phettest. 

The book color is correct in the book representation at the bottom of the screen, 
![screen shot 2017-09-20 at 2 28 47 pm](https://user-images.githubusercontent.com/7852916/30665946-105acb1e-9e10-11e7-8711-8e08578a1a0e.png)

but in the zoomed in view it looks like it's the bright cyan color instead. 
![screen shot 2017-09-20 at 2 28 54 pm](https://user-images.githubusercontent.com/7852916/30665958-15fefc34-9e10-11e7-9c20-129b86bdf061.png)

@oliver-phet @ariel-phet @amyh-phet @terracoda @emily-phet What do you guys think? In particular, what do you think of the black stroke around the zoom window and dashed line, and the gray stroke around the book images- does that work?

**Comment by oliver-phet** [comment]
I like the gray on the books. Just to experiment, I tried making the text gray also and I think it helps the dashed line stand out against the text:
<img width="840" alt="screen shot 2017-09-20 at 2 47 50 pm" src="https://user-images.githubusercontent.com/5090275/30667086-9fa1b028-9e13-11e7-81ae-dc246ee28007.png">

**Comment by oliver-phet** [comment]
But that may not pass contrast tests.

**Comment by zepumph** [comment]
@amanda-phet I'm terribly sorry. You're right about the shades of blue. I actually used three different colors. I ended up grabbing them directly from the picture [here](https://github.com/phetsims/friction/issues/41#issuecomment-330640807) using a chrome extension. I only did it because I was a goof and didn't notice the hex codes above. 

I think we should get on the same page. 
You said:
>The two shades of blue in this mockup are:
#7de2f9 for the book
#00ffff for the atoms

In the picture I see these three colors:
* #79e2fb for the magnified background
* #5cd5f6 for the macro view of the book
* #00ffff for the atoms

What is best?

I am happy to implement anything, sorry for the miscommunication.

**Comment by terracoda** [comment]
@amanda-phet, @oliver-phet, @zepumph, if we make the spines of the books a little thicker to provide more space around the box. And use a darker grey for the book titles we should get more legibility and maybe pass a contrast ratio.

We talked about making the books a little thicker at the meeting on Tuesday. Is that a reasonable thing to do?

**Comment by terracoda** [comment]
With a blue bg of #7de2f9, I would suggest a dark grey like #404040 for the tite text on the spines of the books. This provides a contrast  6.97:1 which is almost 7:1 which is considered high contrast (WCAG AAA) even for small text. 

We can go lighter down to a grey of #595959 providing a contrast of 4.7:1, and still be WCAG AA for small text and AAA for large text.

**Comment by amanda-phet** [comment]
@zepumph color pickers aren't the most reliable, and I didn't realize there were multiple places to enter a color, so please use #7de2f9 for both the magnified [book] background and the macro view of the book. 

Here is a mockup using #595959 for the book titles. 

<img width="778" alt="screen shot 2017-09-21 at 2 51 58 pm" src="https://user-images.githubusercontent.com/7852916/30718209-72ea5c14-9edc-11e7-99c5-c1eed9c9adea.png">


@terracoda I'm not sure about making changes to the actual artwork.. that might be more involved, I don't really know. If it's important for accessibility we should discuss making changes to the design of the sim, but in general I'm assuming we should avoid those kinds of changes because it gets very complicated when we start redesigning a sim. Color changes are super simple, though!

**Comment by terracoda** [comment]
@amanda-phet, totally understand that no change to artwork is the preferred route. If it works, don't fix it! I think the changes, thus far are looking good.

@zepumph and @amanda-phet, I think it would look more natural if the pick focus highlight goes behind the the little magnifier box.

**Comment by zepumph** [comment]
Sounds good to me! I updated the colors in the above commit (both gray and blue). The focusHighlight layering should be resolved in https://github.com/phetsims/scenery/issues/676. 

@amanda-phet what do you think about the wider book spine? I would love a consensus and direction before I implement anything. The way things are set up, it wouldn't just be tweaking a few numbers, but redoing much of the layout. See this picture for what happens when I change the book height from 30 to 40. 
![image](https://user-images.githubusercontent.com/6856943/30838423-ea921386-a217-11e7-9478-bae58f538c44.png)

It would take a bit of work, but not too much. I just want to make just that we want to go down that road. I am also happy to take a stab if people want to see it in action. Let me know!

**Comment by terracoda** [comment]
@amanda-phet, I would love your comments on this. The extra space a thicker spine provides around the title words looks potentially beneficial to me. 

@zepumph and @amanda-phet, I was thinking that the spines on both books would remain consistent, so both would be wider.

**Comment by zepumph** [comment]
From keyboard design meeting today:

* focus highlight needs to go behind the background (invisible outside of the magnified view)
* focus highlight should be behind the magnifiedTarget
* focus highlight around the macro book eroded to make sure that the focusHighlight is never larger than the book bounds.

**Comment by zepumph** [comment]
The changes here were reviewed today. All looks good.

**Comment by zepumph** [comment]
Actually here is a bit more that we decided on. 

* focus highlight a bit higher on top to get around arrows
* focus highlight on top of background row circles

Now all should be ready to go for keyboard nav.
